libs/src/litmus_libs/models.py

```diff
@@ -17,3 +19,12 @@ class DatabaseConfig(BaseModel):
     uris: str
     username: str
     password: str
+
+
+@dataclass
+class TLSConfig:
```

Comment:
```suggestion
class TLSConfigData:
```

Reply:
weak preference, but still.. it's a data structure, "TlsConfig" makes me think of a config file.